app/config/config.yml

```diff
@@ -64,6 +64,10 @@ doctrine:
         auto_generate_proxy_classes: "%kernel.debug%"
         auto_mapping: true
 
+        dql:
```

Comment:
bardzo mało prawdopodobne że format daty zmieni się w bazie + to jest najprostsze rozwiązanie